### PR TITLE
Version 10.21.0

### PR DESCRIPTION
**Pull request by BSaunders95**
Add Dockerfile for node version 10.21.0 build

Required for DVOP-1322

### CODE REVIEW

File: 10.21.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+FROM node:10.21.0-alpine3.11
```
Comment (Jata2k) [inline 10.21.0/Dockerfile]: I would use `10-alpine` as a base so as to avoid pinning ourselves to an older version

**Comment by Jata2k** [comment]
I would change the branch name to `version-10` and I wouldn't merge it into `master`. It can stay in it's own branch and we can delete `master`

File: 10.21.0/Dockerfile (hunk 1 above)
Comment (Jata2k) [inline 10.21.0/Dockerfile]: I'd move these files into the root folder. I don't think we gain anything by having a subfolder

Comment (BSaunders95) [reply 10.21.0/Dockerfile]: Changed 👍

Comment (BSaunders95) [reply 10.21.0/Dockerfile]: Moved 👍

**Pull request closed without merging**